### PR TITLE
Tools: Fix Echo and improve Modprobe

### PR DESCRIPTION
**Pull request by pupacha**
* Echo: enclose file in quotes
* Modprobe: add a new method `load_with_parameters` and support loading kernel
modules with parameters

### CODE REVIEW

File: lisa/tools/modprobe.py [hunk 1]
```diff
@@ -111,6 +111,31 @@ def load(
         )
         return True
 
+    def load_with_parameters(
```
Comment (LiliDeng) [inline lisa/tools/modprobe.py]: add the parameters as option parameter for load method

Comment (pupacha) [reply lisa/tools/modprobe.py]: `load` method can take list of modules as input. Parameters can be passed in `Modprobe` only with one module. Hence, a new method.

Comment (LiliDeng) [reply lisa/tools/modprobe.py]: You can throw an exception if parameters is specified when modules is a list.

Comment (squirrelsc) [reply lisa/tools/modprobe.py]: > `load` method can take list of modules as input. Parameters can be passed in `Modprobe` only with one module. Hence, a new method.

You can just join the mod list and parameters, and let modprobe return non-zero exit-code.

Comment (pupacha) [reply lisa/tools/modprobe.py]: > > `load` method can take list of modules as input. Parameters can be passed in `Modprobe` only with one module. Hence, a new method.
> 
> You can just join the mod list and parameters, and let modprobe return non-zero exit-code.

I dont think we should consciously append paremeters to mod list (`-a` option) and let modprobe fail.


> You can throw an exception if parameters is specified when modules is a list.

Sure, I will raise `UnsupportedOperationException`

**Review by squirrelsc — APPROVED** [review]

**Pull request merged**